docs/src/components/Hero.tsx

```diff
@@ -74,9 +78,10 @@ export function Hero() {
               <p className="fluid-text inline bg-gradient-to-r from-primary via-secondary to-logo bg-clip-text font-display tracking-tight text-transparent ">
                 LaunchFlow Docs
               </p>
-              <p className="mt-3 text-lg tracking-tight text-slate-300 sm:text-xl">
-                Deploy, manage, and share cloud infrastructure across multiple
-                cloud providers and environments using Python.
+              <p className="mt-3 text-lg tracking-tight text-slate-300 sm:text-2xl">
```

Comment:
nit: I feel like this text could be a little smaller

Reply:
I kinda like it taking up the space since its a subtitle. I changed the text a bit, does that change anything?